content/applications/productivity/iot/config/https_certificate_iot.rst

```diff
@@ -0,0 +1,269 @@
+
+.. _https_certificate_iot:
```

Comment:
internal references should have an app name prefix, see https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#prefix-hyperlink-targets-with-application-names